### ISSUE TITLE
format of the classifier data and tflite exporting

### ISSUE THREAD

**Issue by abdelaziz-mahdy** [ultralytics/yolov5#7039]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I want to train a classfication model using classifier.py but dont know the required format 

and will it be possible to convert that model after training to tflite?

**Comment by glenn-jocher** [comment]
@zezo357 👋 Hello! Thanks for asking about **training classification models**. YOLOv5 🚀 classification models are super easy to train and use. The below code clones YOLOv5 classifier [branch](https://github.com/ultralytics/yolov5/tree/classifier), trains a YOLOv5s classifier on the [MNIST](http://yann.lecun.com/exdb/mnist/) dataset for 5 epochs to ~99% accuracy and then runs inference on the first 10 images of the '7' directory.


```python
# Setup
!git clone https://github.com/ultralytics/yolov5 -b classifier  # clone
%cd yolov5
%pip install -qr requirements.txt  # install

# Train
!python classifier.py --model yolov5s --data mnist --epochs 5 --img 128

# Inference
import torch
from classifier import *
model = torch.load('runs/train/exp/weights/best.pt', map_location=torch.device('cpu'))['model'].float()
files = Path('../datasets/mnist/test/7').glob('*.png')  # images from dir
for f in list(files)[:10]:  # first 10 images
    classify(model, size=128, file=f)
```

<img width="1969" alt="Screenshot 2022-03-19 at 21 41 17" src="https://user-images.githubusercontent.com/26833433/159137747-2bc75db8-cdb1-49eb-a8ab-a9c8d76d5c4d.png">

See [classifier.py](https://github.com/ultralytics/yolov5/blob/classifier/classifier.py) usage examples for latest code:
https://github.com/ultralytics/yolov5/blob/d1e2938e2fcfb6faebe74a8b7a680d7959960f2a/classifier.py#L1-L16

Good luck 🍀 and let us know if you have any other questions!

**Comment by abdelaziz-mahdy** [comment]
> @zezo357 👋 Hello! Thanks for asking about **training classification models**. YOLOv5 🚀 classification models are super easy to train and use. The below code clones YOLOv5 classifier [branch](https://github.com/ultralytics/yolov5/tree/classifier), trains a YOLOv5s classifier on the [MNIST](http://yann.lecun.com/exdb/mnist/) dataset for 5 epochs to ~99% accuracy and then runs inference on the first 10 images of the '7' directory.
> 
> 
> ```python
> # Setup
> !git clone https://github.com/ultralytics/yolov5 -b classifier  # clone
> %cd yolov5
> %pip install -qr requirements.txt  # install
> 
> # Train
> !python classifier.py --model yolov5s --data mnist --epochs 5 --img 128
> 
> # Inference
> import torch
> from classifier import *
> model = torch.load('runs/train/exp/weights/best.pt', map_location=torch.device('cpu'))['model'].float()
> files = Path('../datasets/mnist/test/7').glob('*.png')  # images from dir
> for f in list(files)[:10]:  # first 10 images
>     classify(model, size=128, file=f)
> ```
> 
> <img width="1969" alt="Screenshot 2022-03-19 at 21 41 17" src="https://user-images.githubusercontent.com/26833433/159137747-2bc75db8-cdb1-49eb-a8ab-a9c8d76d5c4d.png">
> 
> See [classifier.py](https://github.com/ultralytics/yolov5/blob/classifier/classifier.py) usage examples for latest code:
> https://github.com/ultralytics/yolov5/blob/d1e2938e2fcfb6faebe74a8b7a680d7959960f2a/classifier.py#L1-L16
> 
> Good luck 🍀 and let us know if you have any other questions!

Thanks I will try it 

For the tflite part

I can use pytorch on Android right?
